generate/pkg/generate/generate.go

```diff
@@ -31,6 +31,11 @@ func Generate(filePath, targetPath string) error {
 	}
 	defer aliasFile.Close()
 
+	_, err = io.WriteString(aliasFile, "#!/bin/bash\n")
+	if err != nil {
+		return fmt.Errorf("Warning: could not write shebang", err)
+	}
+
```

Comment:
I don't think this is required. Sourcing the file should be sufficient.

Reply:
This definitely isn't required. I just feel like it's good practice?

Reply:
I believe its completely ignored when the file is sourced. It doesnt hurt anything so Im okay for keeping it.